src/main/java/Duke.java

```diff
@@ -1,10 +1,66 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String HORIZONTAL_LINES = "____________________________________________________________\n";
+    public static final String INDENTATIONS = "    ";
```

Comment:
I think `HORIZONTAL_LINE` and `INDENTATION` might be better names for these two variables.